goal_src/jak1/engine/target/target.gc

```diff
@@ -97,7 +97,9 @@
   )
 
 (defbehavior target-falling-trans target ((arg0 basic) (arg1 time-frame))
-  (if (and (cpad-pressed? (-> self control unknown-cpad-info00 number) circle) (can-feet?))
+  (when (and (cpad-pressed? (-> self control unknown-cpad-info00 number) circle) (can-feet?)
+           (not (logtest? (-> self control unknown-surface00 flags) (surface-flags prevent-attacks-during-launch-jump surf08))) ;; PATCHED - prevent 1 frames
```

Comment:
`prevent-attacks-during-launch-jump` flag is used in some nearby code to decide if you can ground pound during a high jump, figured it made sense to reuse it here